osu.Game/Screens/Edit/Editor.cs

```diff
@@ -169,10 +166,10 @@ private void onModeChanged(EditorScreenMode mode)
             switch (mode)
             {
                 case EditorScreenMode.Compose:
-                    currentScreen = new Compose();
+                    currentScreen = new Compose.Compose();
```

Comment:
unfortunate. for the tournament stuff i've been doing `Name/NameScreen`, could that work better here?